README.md

```diff
@@ -1,17 +1,16 @@
-# Deno Docker
+# deno_docker
 
-Docker files for [deno](https://github.com/denoland/deno) published on Dockerhub:
+Docker files for [Deno](https://github.com/denoland/deno) published on
+Dockerhub:
 
-- Alpine Linux: [hayd/alpine-deno](https://hub.docker.com/r/hayd/alpine-deno/) (\~28Mb)
-- Centos: [hayd/centos-deno](https://hub.docker.com/r/hayd/centos-deno/) (\~94Mb)
-- Debian: [hayd/debian-deno](https://hub.docker.com/r/hayd/debian-deno/) (\~47Mb)
-- Distroless: [hayd/distroless-deno](https://hub.docker.com/r/hayd/distroless-deno) (\~28Mb)
-- Ubuntu: [hayd/ubuntu-deno](https://hub.docker.com/r/hayd/ubuntu-deno/) (\~50Mb)
+- Alpine Linux: [denoland/deno:alpine](https://hub.docker.com/r/denoland/deno)
+  (default)
```

Comment:
Can we make something else the default, like Debian or Ubuntu?

Reply:
but they so big :-)

Reply:
the latest stable image for ubuntu is 72.7Mb, for a Docker image that's fine imo, and Alpine is just such a pain to work with, I'd rather have something that's nicer and more user friendly as the defaults, if people want to optimize they can pull the smaller image for their prod build (that's also how the Go Docker image is set up)

Reply:
Debian is 114Mb and CentOS is 209Mb for reference

Reply:
IMO debian-slim makes a lot of sense since it has a slightly cleaner stable rolling release vs ubuntu... (I think debian _slim_ is 72mb ?)

I am never sure what the cool kids are using these days...

Reply:
yeah, the `stable-slim` tag is pretty neat